grails-app/controllers/org/pih/warehouse/api/StockMovementItemApiController.groovy

```diff
@@ -35,6 +35,29 @@ class StockMovementItemApiController {
         render([data: stockMovementItem] as JSON)
     }
 
+    def getStockMovementItems = {
+        List<StockMovementItem> stockMovementItems = stockMovementService.getStockMovementItems(params.id, params.startIndex, params.stopIndex)
+        render([data: stockMovementItems] as JSON)
```

Comment:
@DominikaHatala I think it would be handy to get in response current pageSize, offset and totalCount (or at least totalCount as would PagedResultList do)
@jmiranda what do you think about it?

Reply:
@awalkowiak Yes, agreed. I also don't love start and stop index. This puts a lot of undue pressure on the API client to manage all of this information. For now, until I have a chance to define a Grails-agnostic API paging mechanism let's stick with the Grails conventions of using max, offset, totalCount. 

Eventually (just so it's clear) I want to be able to apply the same conventions for paging/filtering/sorting across all of our REST APIs and allow clients to use both request headers and parameters to pass this information along to the endpoint.